bin/migrate-oats-data/planning_review/decisions/init_planning_review_decisions.py

```diff
@@ -0,0 +1,188 @@
+from common import (
+    BATCH_UPLOAD_SIZE,
+    OATS_ETL_USER,
+    setup_and_get_logger,
+    add_timezone_and_keep_date_part,
+    OatsToAlcsDecisionOutcomes,
+    AlcsPlanningReviewOutcomes,
+)
+from db import inject_conn_pool
+from psycopg2.extras import RealDictCursor, execute_batch
+from datetime import datetime
+
+etl_name = "init_planning_review_decision"
+logger = setup_and_get_logger(etl_name)
+
+
+@inject_conn_pool
+def process_planning_review_decisions(conn=None, batch_size=BATCH_UPLOAD_SIZE):
+    """
+    This function is responsible for populating alcs.planning_referral in ALCS.
+
+    Args:
+    conn (psycopg2.extensions.connection): PostgreSQL database connection. Provided by the decorator.
+    batch_size (int): The number of items to process at once. Defaults to BATCH_UPLOAD_SIZE.
+    """
+
+    logger.info("Start insert planning decision fields")
+    with conn.cursor(cursor_factory=RealDictCursor) as cursor:
+        with open(
+            "planning_review/sql/decisions/planning_review_decisions_insert_count.sql",
+            "r",
+            encoding="utf-8",
+        ) as sql_file:
+            count_query = sql_file.read()
+            cursor.execute(count_query)
+            count_total = dict(cursor.fetchone())["count"]
+        logger.info(f"Total Planning decision data to insert: {count_total}")
+
+        failed_inserts_count = 0
+        successful_inserts_count = 0
+        last_planning_review_id = 0
+
+        with open(
+            "planning_review/sql/decisions/planning_review_decisions_insert.sql",
+            "r",
+            encoding="utf-8",
+        ) as sql_file:
+            query = sql_file.read()
+            while True:
+                cursor.execute(
+                    f"""
+                        {query} 
+                        WHERE opd.planning_decision_id > {last_planning_review_id} ORDER BY opd.planning_decision_id;
+                    """
+                )
+
+                rows = cursor.fetchmany(batch_size)
+
+                if not rows:
+                    break
+                try:
+                    inserted_data = _insert_base_fields(conn, batch_size, cursor, rows)
+
+                    successful_inserts_count = successful_inserts_count + len(
+                        inserted_data
+                    )
+                    last_planning_review_id = dict(inserted_data[-1])[
+                        "planning_decision_id"
+                    ]
+
+                    logger.debug(
+                        f"Retrieved/updated items count: {len(inserted_data)}; total successfully inserted planning referral so far {successful_inserts_count}; last updated planning_decision_id: {last_planning_review_id}"
+                    )
+                except Exception as err:
+                    # this is NOT going to be caused by actual data insert failure. This code is only executed when the code error appears or connection to DB is lost
```

Comment:
This is not true, since you are performing insert without on conflict do nothing